src/ducks/balance/AccountsList.jsx

```diff
@@ -97,4 +97,7 @@ AccountsList.propTypes = {
   onSwitchChange: PropTypes.func
 }
 
-export default AccountsList
+const MemoizedAccountsList = React.memo(props => {
+  return <AccountsList {...props} />
+})
+export default MemoizedAccountsList
```

Comment:
nit: pourquoi pas simplement `export default React.memo(AccountsList)` ? A cause des tests ?

Ou sinon `const MemoizedAccountsList = React.memo(AccountsList)` qui est pareil non ? 🤔 (pas forcément un changement à faire, c'est que ça me fait douter du coup...)

Reply:
All you said is the same thing. I write it that way because I can easily add a log or add a 2nd arg in order to test few methods to check equality